Dockerfile

```diff
@@ -18,6 +22,9 @@ EXPOSE 8080
 
 USER ${USERMAP_UID}
 RUN yarn install --quiet
+RUN yarn add braces@3.0.3
```

Comment:
This and the next line of code must go in package.json and in that if this is a dev dependency then it must be under that.

Reply:
@hemantgoswami braces is now added to packages.json and pushed updated changes to branch.